packages/views/screens/managers/request/index.js

```diff
@@ -1,41 +1,11 @@
-import React from 'react';
 import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
 import { withTranslation } from 'react-i18next';
-import { tokenKeys } from '@token/fungible/consts/tokens';
-import RequestBtc from './requestBtc';
-import RequestLsk from './requestLsk';
-
-const TagNameMap = {
-  LSK: RequestLsk,
-  BTC: RequestBtc,
-};
-
-const Request = ({
-  account, t, token,
-}) => {
-  const TagName = TagNameMap[token];
-  const address = account.info ? account.info[token].summary.address : '';
-  return <TagName address={address} t={t} />;
-};
-
-Request.propTypes = {
-  account: PropTypes.object,
-  token: PropTypes.oneOf(tokenKeys).isRequired,
-  t: PropTypes.func.isRequired,
-};
-
-Request.defaultProps = {
-  account: {},
-  token: 'LSK',
-  /* istanbul ignore next */
-  t: key => key,
-};
+import { selectActiveTokenAccount } from '@common/store';
+import Request from './request';
 
 export default connect(
   state => ({
-    token: state.token.active,
-    account: state.wallet,
+    address: selectActiveTokenAccount(state).summary?.address ?? '',
```

Comment:
```suggestion
    address: selectActiveTokenAccount(state)?.summary?.address ?? '',
```

Reply:
The active token returned by `selectActiveTokenAccount` can never be `undefined`.